### DISCUSSION TITLE
Problème de lancement

### DISCUSSION THREAD

**Discussion by Phoenixlafoudre** [Kodi-vStream/venom-xbmc-addons#4010] [Support & aide]
Bonjour
J ai fais installer à une ami openelec et kodi pour ensuite lancé vstream mais elle n arrive pas à lancée de lien je lui ai demandé les logs ils sont en pièce jointe 
![received_505332915080789](https://user-images.githubusercontent.com/124449234/216774540-436b2fba-9321-49a8-9ca7-cd45539e0b63.jpeg)
![received_505332915080789](https://user-images.githubusercontent.com/124449234/216774552-0b4a08a2-3a7c-4979-89c7-dda3e9434a6a.jpeg)
![received_1244964796092679](https://user-images.githubusercontent.com/124449234/216774560-ab9870c0-6624-431a-b10d-65c3b16f541d.jpeg)
![received_1317425068825602](https://user-images.githubusercontent.com/124449234/216774566-793ad9b1-c892-418f-9118-3dc9b5bda1f9.jpeg)
![received_733005988435068](https://user-images.githubusercontent.com/124449234/216774571-d1a062cd-da02-4de6-8e90-1761a658aa25.jpeg)
![received_425497586411362](https://user-images.githubusercontent.com/124449234/216774575-15d30d34-d2f2-4ed4-a1e3-8e0902cb4601.jpeg)
Auriez-vous une idée de ce que je peux faire pour l aider ?
Merci d avance

**Comment by yodidon** [comment]
Tu as essayé différentes sources ?

**Reply by Phoenixlafoudre** [reply]
Oui elle a essayé rien de concluant

**Comment by TmpName** [comment]
Ca ressemble a des probleme de connexions, pas de soucis pour arriver jusqu'au lien mais ca plante apres.
Il y a un navigateur web sur sa machine ? Si oui dis lui de tester un site en direct.

**Comment by Phoenixlafoudre** [comment]
Bonjour le problème s est résolu aujourd'hui.
Merci beaucoup pour vos retours

**Reply by sizanic** [reply]
Bonjour,
On a sortie une nouvelle version cette nuit, il y a peut-être un lien avec le bon fonctionnement 🤔
